VersionControlUI/src/items/VDiffComparisonPair.h

```diff
@@ -53,15 +54,20 @@ class VERSIONCONTROLUI_API VDiffComparisonPair : public Super<Visualization::Ite
 		static void initializeForms();
 		virtual bool isSensitiveToScale() const override;
 		virtual void determineChildren() override;
+		int determineForm() override;
```

Comment:
it is not strictly needed, but please put `virtual` infront here, to match the style of the rest of envision.